src/sql/workbench/browser/designer/designer.ts

```diff
@@ -836,6 +824,53 @@ export class Designer extends Disposable implements IThemable {
 					headerRowHeight: TableHeaderRowHeight,
 					editorLock: new Slick.EditorLock()
 				});
+
+				let moveRowsPlugin = new RowMoveManager({
```

Comment:
1. this block should be executed when table is configured to allow move.
2. use `const` instead of `let` whenever possible.
3. remove the blank lines.